force-app/main/default/classes/NextStepFuture.cls

```diff
@@ -27,9 +27,22 @@ public with sharing class NextStepFuture {
      * Opportunity Next Step field is a standard field
      * Update the corresponding test class AsynchronousTest and test method nextStepFuture_testUpdateAllNextStep to cover test this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    @future
+    public static void updateAllNextStep () {
```

Comment:
Trying to update everything in the database here will cause issues with large data volumes. Limits are greater with @future, but there are still limits. Can you update this to pass in just the records that you inserted (in the test). Part of the question is having you learn what can be passed into @future methods. The rest of the method is good, just needs to be limited records